packages/main/src/themes/BusyIndicator.css

```diff
@@ -76,13 +76,16 @@
 	border-radius: inherit;
 }
 
-.ui5-busy-indicator-busy-area {
+:host(:not(:empty)) .ui5-busy-indicator-busy-area {
```

Comment:
Empty is not the best selector because you might have html comment inside the busy indicator and it still fail.

Check willShowContent util as suggestion to workaround it.